src/python/pants/backend/python/rules/pytest_runner.py

```diff
@@ -275,16 +278,12 @@ async def run_python_test(
 
 
 @rule(desc="Run Pytest in an interactive process")
-async def debug_python_test(test_setup: TestTargetSetup) -> TestDebugRequest:
+def debug_python_test(setup: TestTargetSetup) -> TestDebugRequest:
```

Comment:
async?

Reply:
No need for async if there are no `await`s in the rule.

Reply:
Feels like a gotcha, if someone adds `await` they will have to notice this and add `async`.  Although that's just a normal demand of asyncio, so maybe it's fine.

Yeah, it's fine.